src/appleseed/renderer/kernel/rendering/final/adaptivetilerenderer.cpp

```diff
@@ -0,0 +1,521 @@
+
+//
+// This source file is part of appleseed.
+// Visit https://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2010-2013 Francois Beaune, Jupiter Jazz Limited
```

Comment:
Fix copyright notice. For files you created, there should be a single notice line with text

    Copyright (c) 2018 Kevin Masson, The appleseedhq Organization

Reply:
Ok :)